instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/dynamodbattributes.go

```diff
@@ -15,7 +15,14 @@ import (
 )
 
 // DynamoDBAttributeSetter sets DynamoDB specific attributes depending on the DynamoDB operation being performed.
+//
+// Deprecated: Use DynamoDBAttributeBuilder instead. This will be removed in a future release.
```

Comment:
This is not mentioned in changelog.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6543/commits/231e92b01921f58ff0d10ba02324684c7e2e5134